### PR TITLE
tests: Fix codec specific capabilities

### PR DESCRIPTION
**Pull request by MariuszSkamra**
This fixes exposing capabilities instead of configuration in PACS characteristic.

### CODE REVIEW

File: samples/bluetooth/unicast_audio_server/src/main.c [hunk 1]
```diff
@@ -29,8 +29,12 @@
 				  BT_AUDIO_CONTEXT_TYPE_MEDIA | \
 				  BT_AUDIO_CONTEXT_TYPE_GAME)
```
Comment (Thalley) [inline samples/bluetooth/unicast_audio_server/src/main.c:31]: The commit message is off here, as this is a sample and not a test :)

**Review by Thalley — CHANGES_REQUESTED** [review]
Any plans on updating the remaining uses of the preset macros used in e.g. the shell? 

Does broadcast also need this change?

**Comment by MariuszSkamra** [comment]
> Any plans on updating the remaining uses of the preset macros used in e.g. the shell?
> 
> Does broadcast also need this change?

Looks like the shell implementation is correct

**Review by asbjornsabo — COMMENTED** [review]
I find the commit messages unclear:
"This fixes exposing capabilities instead of configuration in PACS characteristic."

Does this mean that the code used to expose capabilities, not configuration, and these commits change that?  
Or does it mean the opposite, the code used to expose configuration, but has been changed to expose capabilities?

File: samples/bluetooth/unicast_audio_server/src/main.c [hunk 2]
```diff
@@ -29,8 +29,12 @@
...
 
-/* Mandatory support preset by both client and server */
-static struct bt_audio_lc3_preset preset_16_2_1 = BT_AUDIO_LC3_UNICAST_PRESET_16_2_1;
+#define CHANNEL_COUNT_1 BIT(0)
```
Comment (Thalley) [inline samples/bluetooth/unicast_audio_server/src/main.c:32]: Ideally we'll provide helper functions to help format things like this, but this is fine for now

**Review by Thalley — APPROVED** [review]

**Review by asbjornsabo — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**